test/test.jl

```diff
@@ -1,16 +1,28 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-using Base.Test
```

Comment:
@iainnz added this one in https://github.com/JuliaLang/julia/commit/dc4f1798fe2fa7429e2551caba529cf0b022150d. given he probably won't notice it's gone I don't feel strongly enough about it to add it back here, but @kshyatt has definitely added several of these around the linalg testing and elsewhere